source/views.txt

```diff
@@ -0,0 +1,106 @@
+.. _view-details:
+
+=====
+Views
+=====
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Views are read-only results of an
```

Comment:
@nlarew added a blurb here for View use cases. If you think this can be improved / if there's anything else you think we should highlight here lmk.